portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,58 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="author" content="Essence Cain">
+    <meta name="description" content="Essence Cain's portfolio page">
+    <meta name="keywords" content="Essence Cain, portfolio, NCAT">
+    <title>Essence Cain's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <ul class="navbar">
+          <li><a href="default.asp">Home</a></li>
```

Comment:
Can you update the indentation of this file? Some of them are 2 spaces and some are 4 spaces. I would go with 2 spaces.